src/main/java/org/jpeek/calculus/java/Ccm.java

```diff
@@ -52,14 +63,19 @@ public XML node(
                 ).toString()
             );
         }
-        return Ccm.withFixedNcc(
-            new XSLDocument(
-                new UncheckedInput(
-                    new ResourceOf("org/jpeek/metrics/CCM.xsl")
-                ).stream()
-            ).transform(skeleton),
-            skeleton
+        final XSLDocument doc = new XSLDocument(
+            new UncheckedText(
+                new TextOf(
+                    new ResourceOf(
+                        new FormattedText("org/jpeek/metrics/%s.xsl", metric)
+                    )
+                )
+            ).asString(),
+            Sources.DUMMY,
```

Comment:
not clear why do we need to apply xsl for dymmy sources, while ccm contains 5 columns for now and we have a problem with 2 of them only 
https://user-images.githubusercontent.com/63350301/117303386-4ff0da00-ae85-11eb-82c3-869f1a66521d.png